Qt/mainwindow.h

```diff
@@ -90,27 +90,37 @@ private slots:
 	void takeScreen() { g_TakeScreenshot = true; }
 	void consoleAct();
 
-	// Options
-	// Core
-	void vertexDynarecAct() { g_Config.bVertexDecoderJit = !g_Config.bVertexDecoderJit; }
-	void fastmemAct() { g_Config.bFastMemory = !g_Config.bFastMemory; }
-	void ignoreIllegalAct() { g_Config.bIgnoreBadMemAccess = !g_Config.bIgnoreBadMemAccess; }
-
-	// Video
-	void anisotropicGroup_triggered(QAction *action) { g_Config.iAnisotropyLevel = action->data().toInt(); }
+	// Game settings
+	void keepOnTopAct() { g_Config.bTopMost = !g_Config.bTopMost; }
+	void pauseWhenNotFocusedAct() { g_Config.bPauseOnLostFocus = !g_Config.bPauseOnLostFocus; }
+	void languageAct() { NativeMessageReceived("language screen", ""); }
+	void controlMappingAct() { NativeMessageReceived("control mapping", ""); }
+	void displayLayoutEditorAct() { NativeMessageReceived("display layout editor", ""); }
+	void moreSettingsAct() { NativeMessageReceived("settings", ""); }
 
 	void bufferRenderAct() {
 		g_Config.iRenderingMode = !g_Config.iRenderingMode;
 		NativeMessageReceived("gpu_resized", "");
 	}
 	void linearAct() { g_Config.iTexFiltering = (g_Config.iTexFiltering != 0) ? 0 : 3; }
 
-	void screenGroup_triggered(QAction *action) { SetWindowScale(action->data().toInt()); }
+	void postprocessingShaderGroup_triggered(QAction *action) { g_Config.sPostShaderName = action->data().toInt(); }
+	void renderingResolutionGroup_triggered(QAction *action) { g_Config.iInternalResolution = action->data().toInt(); }
+	void windowGroup_triggered(QAction *action) { SetWindowScale(action->data().toInt()); }
 
 	void displayLayoutGroup_triggered(QAction *action) {
 		g_Config.iSmallDisplayZoomType = action->data().toInt();
 		NativeMessageReceived("gpu_resized", "");
 	}
+	void renderingModeGroup_triggered(QAction *action) { g_Config.iRenderingMode = action->data().toInt(); }
+	void autoframeskipAct() { g_Config.bAutoFrameSkip = !g_Config.bAutoFrameSkip; }
```

Comment:
If `g_Config.iRenderingMode` is `FB_NON_BUFFERED_MODE`, needs to change it to FB_BUFFERED_MODE` and issue `NativeMessageReceived("gpu_resized", "");`.  Otherwise auto frameskip won't work.

-[Unknown]